```
/Users/ec2-user/jenkins/workspace/swift-PR-macos-smoke-test/branch-main/swift/test/SourceKit/Refactoring/shorthand_shadow.swift:35:14: error: BUILTIN: expected string not found in input
 // BUILTIN: error: cannot rename system symbol 'file'
             ^
<stdin>:1:1: note: scanning from here
{
^
<stdin>:14:102: note: possible intended match here
error response (Request Failed): SOURCE_DIR/test/SourceKit/Refactoring/shorthand_shadow.swift:31:10: error: cannot rename system symbol 'file()'
```

@DougGregor It looks like this also affects the built-in `#file`/`#line`/etc in diagnostics like this. Is that acceptable? If so i can update this test too.

--- a/test/SourceKit/Refactoring/shorthand_shadow.swift
+++ b/test/SourceKit/Refactoring/shorthand_shadow.swift
@@ -32,7 +32,7 @@ func renameBuiltinMacroWithoutHash() {
     // RUN: not %sourcekitd-test -req=find-local-rename-ranges -pos=%(line+1):9 %s -- %s 2>&1 | %FileCheck %s --check-prefix=BUILTIN
     _ = file
   }
-  // BUILTIN: error: cannot rename system symbol 'file'
+  // BUILTIN: error: cannot rename system symbol 'file()'
 }
 
 

